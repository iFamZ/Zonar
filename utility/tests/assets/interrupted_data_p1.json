{
    "type": "interrupted data part 1 - ends with latitude - needs longitude",
    "data": [
        {
            "ts": 1650500050,
            "k": "latitude",
            "v": "47.608013"
        },
        {
            "ts": 1650500050,
            "k": "longitude",
            "v": "-122.335167"
        },
        {
            "ts": 1650500050,
            "k": "speed",
            "v": "55.15"
        },
        {
            "ts": 1650500051,
            "k": "latitude",
            "v": "47.608013"
        },
        {
            "ts": 1650500051,
            "k": "longitude",
            "v": "-122.335160"
        },
        {
            "ts": 1650500052,
            "k": "latitude",
            "v": "47.608049"
        }
    ]
}