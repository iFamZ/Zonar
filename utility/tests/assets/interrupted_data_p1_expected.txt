{'latitude': '47.608013', 'longitude': '-122.335167', 'speed': None, 'ts': 1650500050}
{'latitude': '47.608013', 'longitude': '-122.335160', 'speed': '55.15', 'ts': 1650500051}
